test/lint/README.md

```
@@ -17,6 +17,29 @@ docker run --rm -v $(pwd):/bitcoin -it bitcoin-linter
 After building the container once, you can simply run the last command any time you
 want to lint.
 
+#### Dependencies
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)